src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturoteca.controllers;
+
+import java.util.*;
+
+import culturoteca.exception.VideoNotFoundException;
+import culturoteca.model.Video;
+import culturoteca.service.impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+	private final CultureMediaServiceImpl cultureMediaService;
```

Comment:
On lines 11 and 14, remove CultureMediaSeriveImpl from CultureMediaService since the CultureMediaController and CultureMediaSeriveImpl classes do not have direct contact.